src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -33,7 +33,7 @@ public void OverflowOnFluentCall()
                     numberFluentCalls = 460;
                     break;
                 case 32 when !isDebug:
-                    numberFluentCalls = 1300;
+                    numberFluentCalls = 1100;
```

Comment:
>1100 [](start = 40, length = 4)

Filed [https://github.com/dotnet/roslyn/issues/33775](https://github.com/dotnet/roslyn/issues/33775) to investigate further and restore our expectations